packages/widget-chat/src/components/title-bar/index.js

```diff
@@ -0,0 +1,28 @@
+import React, {Component, PropTypes} from 'react';
+import classNames from 'classnames';
+
+import styles from './styles.css';
+
+import Avatar from '../avatar';
+
+class TitleBar extends Component {
```

Comment:
can this be done with a stateless component? I think the only reason eslint didn't encourage you switch to stateless is because there's a shouldComponentUpdate (which, as I understand it, is less efficient than stateless functions when coupled with redux).